lighthouse/lighthouserc.js

```diff
@@ -0,0 +1,100 @@
+const AUDIT_URLS = {
```

Comment:
**nitpick:** is it worth moving this to a seperate file - like we keep the Cypress config separate from the tests? It could still live within this folder - here's an example https://github.com/bbc/simorgh/blob/latest/3rdPartyCypress/cypress/integration/prerollAds/config.js#L24

Reply:
I like that the config is all in one file without having to jump between files. I'm not a huge fan of splitting code up prematurely but also not too fussy if you would rather it be separate.

Reply:
Yeah, let's leave it as is - no point in optimising too early